src/crypto/index.ts

```diff
@@ -261,6 +266,7 @@ export enum CryptoEvent {
      */
     VerificationRequestReceived = "crypto.verificationRequestReceived",
 
+    /** @deprecated Event not fired by the rust crypto */
     Warning = "crypto.warning",
     WillUpdateDevices = "crypto.willUpdateDevices",
```

Comment:
I wonder if we should deprecate this one while we're here. Even though it is fired by rust crypto, it's a bit of a bodge, and means exactly the same as "DevicesUpdated" under rust crypto.